app/src/main/assets/ime/text/characters/extended_popups/tr.json

```
@@ -1,99 +1,31 @@
 {
   "type": "characters/extended_popups",
   "name": "tr",
-  "authors": [ "kisekinopureya", "patrickgold" ],
+  "authors": [ "dvrnynr", "kisekinopureya", "patrickgold" ],
   "mapping": {
     "all": {
-      "a": {
-        "relevant": [
-          { "$": "auto_text_key", "code":  226, "label": "â" },
-          { "$": "auto_text_key", "code":  228, "label": "ä" },
-          { "$": "auto_text_key", "code":  225, "label": "á" }
-        ]
-      },
       "c": {
-        "main": { "$": "auto_text_key", "code":  231, "label": "ç" },
-        "relevant": [
-          { "$": "auto_text_key", "code":  269, "label": "č" },
-          { "$": "auto_text_key", "code":  263, "label": "ć" }
-        ]
-      },
-      "e": {
-        "relevant": [
-          { "$": "auto_text_key", "code":  233, "label": "é" },
-          { "$": "auto_text_key", "code":  601, "label": "ə" },
-          { "$": "auto_text_key", "code":  234, "label": "ê" }
-        ]
+        "main": { "$": "auto_text_key", "code":  231, "label": "ç" }
       },
       "g": {
         "main": { "$": "auto_text_key", "code":  287, "label": "ğ" }
       },
       "i": {
-        "main": { "$": "auto_text_key", "code":  305, "label": "ı" },
-        "relevant": [
-          { "$": "auto_text_key", "code":  303, "label": "į" },
-          { "$": "auto_text_key", "code":  236, "label": "ì" },
-          { "$": "auto_text_key", "code":  237, "label": "í" },
-          { "$": "auto_text_key", "code":  299, "label": "ī" },
-          { "$": "auto_text_key", "code":  238, "label": "î" },
-          { "$": "auto_text_key", "code":  239, "label": "ï" }
-        ]
-      },
+        "main": { "$": "case_selector", "lower": { "code":  105, "label": "i" },
+                                        "upper": { "code":  304, "label": "İ" }
+      }},
```

It looks like here is a bracket too much.
```suggestion
      },
```